### ISSUE TITLE
CUDA 12 Not Supported

### ISSUE THREAD

**Issue by yavuz650** [accel-sim/accel-sim-framework#178]
I followed the instructions [here](https://github.com/accel-sim/accel-sim-framework/tree/dev/util/tracer_nvbit) to extract traces from an individual application, but when I run the command `LD_PRELOAD=./tracer_tool/tracer_tool.so ./nvbit_release/test-apps/vectoradd/vectoradd`, the program seems to run indefinitely. No traces are generated, and the program just hangs.

Any clues?

This is on Ubuntu 22.04 with Cuda toolkit 12.0.

**Comment by yavuz650** [comment]
Looks like the tracer doesn't work with Cuda 12.0. It works fine with older versions.

**Comment by JRPan** [comment]
Okay, thank you for reporting that. The NVBit tool is not updated yet. There is nothing we can do. I don't think it supports 40s cards either. Let's keep this issue open until the NVBit is updated

JR

**Comment by barnes88** [comment]
Nvbit is updated (and the tracer tool has been updated too). I think we can close this

**Comment by tgrogers** [comment]
OK great - this is all in dev now?

**Comment by barnes88** [comment]
Yes https://github.com/accel-sim/accel-sim-framework/pull/321#ref-issue-2394259392
